### PR TITLE
first draft of 2024 script

### PR DESCRIPTION
**Pull request by troy-halpin**

### CODE REVIEW

File: 2024/script.txt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+C: It sure does, but speaking of 2023 that brings us to our top story.
+
+[cut to solo medium shot of christian with dramatic pivot to secondary camera]
+C: In what may feel like a scene straight out of classic 1993 film Groundhog Day, the UIF roadmap for 2024 is looking 
+alarmingly similar to the initial proposal for the 2023 roadmap.
+C: There have been a “growth” of concerns around the similarity, leaving some “division” among UIF members. 
+C: Some are questioning if this was a foreseeable and preventable outcome, while Friedrich Nietzsche maintains 
+that time is a flat circle and we are doomed to endlessly repeat our actions
+T: [Off-camera] Nietzsche? Who let him on the team?
+C: Shut up Troy, this is my piece // can workshop interjection
```
Comment (troy-halpin) [inline 2024/script.txt]: the goal is to later make a callback to "this is my piece" with a play on words. but i don't know if "shut up troy" or something like "hey now, this is my piece" is better. we can play around with delivery

File: 2024/script.txt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+-- Used Component Salesman ad --
+
+T: Hello and welcome back. 
+T: [Massive grin, holding back laughter] It is with extreme sadness I must report the passing of Core Labs
+C: *gasp*
+T: [still grinning] Yes, unfortunately, the much beloved and never problematic monorepo has finally bitten the dust
+T: The funeral cake is still fresh in fact [eat cake]
+C: Wait where did you get cake, I want some
+T: shut up christian, this is my piece
+T: [mouth somewhat full of cake] The celebration of life has been carried out in several states, countries, and time
```
Comment (chrisarevir) [inline 2024/script.txt]: Dancing Coffin with core-labs written on it

File: 2024/script.txt (hunk 1 above)
Comment (troy-halpin) [reply 2024/script.txt]: decided not to do this as we took out the later callback

File: 2024/script.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+who played Captain Jack Sparrow in the previous films is attached to the project, but our sources say the Sidepanel is.
+
+
+C: And now for our final segment, we have Brock from Procore HQ. How's the office?
+
+-- Brock office --
+
+T: And that was Procorelitzer prize winning journalist, Brock Noah.
+
+C: That's all of our stories for the day, thank you for watching.
```
Comment (chrisarevir) [inline 2024/script.txt:85]:
```suggestion
C: That's all of our story points for the day, thank you for watching.
```

**Pull request closed without merging**